graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/CodeGen.kt

```diff
@@ -472,6 +472,7 @@ data class CodeGenConfig(
     val generateInterfaceSetters: Boolean = true,
     val includeImports: Map<String, String> = emptyMap(),
     val includeEnumImports: Map<String, Map<String, String>> = emptyMap(),
+    val includeClassImports: Map<String, Map<String, String>> = emptyMap(),
```

Comment:
Can you check if it is possible to use typemapping instead?

Reply:
As discussed, typeMapping will require complex changes and includeImports will not support same class names from different packages. includeClassImports is the next best approach, but will need additional story for deweyjose repo.

Reply:
Makes sense. Thanks for looking into it.